### DISCUSSION TITLE
How do I actually terminate the text input?

### DISCUSSION THREAD

**Discussion by calebtt** [rhasspy/piper#273] [Q&A] [answered]
Sorry if I'm a bit of a newbie when it comes to this, I try to use command line tools as little as possible in my professional life working with mostly Windows and Visual Studio + C# and C++.

But once I get the stdin reading in my text, how do I actually end the text input and create the output audio?

Running the compiled .exe with powershell via:
`./piper -m "K:\dev\Piper_Voices\libritts_r\en_US-libritts_r-medium.onnx" -f "output.wav"`

It enters what appears to be a line by line text input mode that I don't know how to end without closing the program.

**Comment by synesthesiam** [comment]
Usually CTRL+D

**Reply by calebtt** [reply]
Ehh unfortunately that isn't working.

**Reply by calebtt** [reply, marked as answer]
Found it, Ctrl+Z for my powershell terminal.
